### PR TITLE
MES-59 - Added terminate page navigation from test report

### PR DESCRIPTION
**Pull request by roberthall12**
## Description and relevant Jira numbers

Added navigation from the test report 'End Test' modal to the terminate version of the debrief page.
https://jira.i-env.net/browse/MES-59

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [ ] branch rebased against the latest develop
- [ ] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [ ] Any new reusable component/widget added to component library on Confluence

- [ ] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [ ] PO's approval
Screen Shot 2019-05-02 at 11 30 09

### CODE REVIEW

File: src/pages/test-report/test-report.ts [hunk 1]
```diff
@@ -186,11 +186,26 @@ export class TestReportPage extends BasePageComponent {
...
+  }
+
+  onContinue = (): void => {
+    this.modal.dismiss()
+    .then(() => this.navCtrl.push('DebriefPage', { outcome: 'pass' }));
+  }
+
+  onTerminate = (): void => {
+    this.modal.dismiss()
+    // TODO - MES-59 to handle terminate test page
```
Comment (ammarhaiderbjss) [inline src/pages/test-report/test-report.ts]: comment can be removed?

File: src/pages/test-report/test-report.ts [hunk 2]
```diff
@@ -186,11 +186,26 @@ export class TestReportPage extends BasePageComponent {
         break;
       case ModalEvent.TERMINATE:
         // TODO - MES-59 to handle terminate test page
```
Comment (ammarhaiderbjss) [inline src/pages/test-report/test-report.ts]: same here?

File: src/pages/test-report/test-report.ts (hunk 1 above)
Comment (roberthall12) [reply src/pages/test-report/test-report.ts]: Removed

File: src/pages/test-report/test-report.ts (hunk 2 above)
Comment (roberthall12) [reply src/pages/test-report/test-report.ts]: removed

**Pull request merged**